packages/cadl-ranch-specs/http/payload/multipart/mockapi.ts

```diff
@@ -98,3 +123,7 @@ Scenarios.Payload_MultiPart_FormData_withMultiBinaryParts = passOnSuccess(
 Scenarios.Payload_MultiPart_FormData_withJsonArrayParts = passOnSuccess(
   createMockApis("json-array-parts", [checkPreviousAddresses, checkProfileImage]),
 );
+
+Scenarios.Payload_MultiPart_FormData_withPureMultiBinaryParts = passOnSuccess(
```

Comment:
You may want to use `passByKeys` to test 2 scenario on this API.
https://github.com/Azure/cadl-ranch/blob/82d302643fac9c1192a9ac7ecdfb308e4b4cebed/packages/cadl-ranch-specs/http/payload/content-negotiation/mockapi.ts#L7

One is user provides `picture` file; another is user does not provide `picture` file. (e.g. `pass: "multiBinaryOptionalNull` and `pass: "multiBinaryOptionalNonNull`)

Reply:
updated